course/include/isometry.h

```diff
@@ -117,4 +120,49 @@ inline std::ostream &operator<<(std::ostream &ss, const Matrix3 &vec) {
             << vec.row(2)[2] << "]]";
 }
 
+class Isometry {
+public:
+  Isometry() {
+    rotation_ = Matrix3::kIdentity;
+    translation_ = Vector3::kZero;
+  }
+  Isometry(const Vector3 &trans, const Matrix3 &rot) {
+    translation_ = trans;
+    rotation_ = rot;
+  }
+
+  static const Isometry FromTranslation(const Vector3 &vec);
+
+  Vector3 operator*(const Vector3 &rhs) const {
+    return (rotation_.product(rhs) + translation_);
+  }
+  Isometry operator*(const Isometry &rhs) const {
+    return (Isometry{rotation_.product(rhs.translation_) + translation_,
+                     (rotation_.product(rhs.rotation_))});
+  }
+  Isometry inverse() const {
+    // Only does inverse of translation (do we need inverse of rotation?)
```

Comment:
Yes, you do. An isometry multiplied by its inverse should take you back to the starting point.

Reply:
Done